### ISSUE TITLE
Oops... Mechanical and Electrical CAD issue

### ISSUE THREAD

**Issue by JasonFischell** [JasonFischell/FischellFancyFlashLight#3]
Sorry. Both are done now. I didn't realize we were supposed to create issues for some reason. My bad.

**Comment by ps178** [comment]
Hi Jason,

You have a lot of files under your Milestone 5 folder. Could you more clearly label which file is your final case CAD to be reviewed :)

**Comment by JasonFischell** [comment]
This has been revised since then. Let me make it a little clearer. But first, do you want drawings or just CAD files?

**Comment by mlp6** [comment]
@JasonFischell looking good... I will let @ericsrichardson comment on the finer aspects of this design too, but from my perspective this is shaping up nicely.  I'm a little concerned that our printers may not capture some of the smaller features well, specifically some of the insets that might get filled with some supports.

**Comment by JasonFischell** [comment]
Dr. Palmeri,

I have already printed 4 different generations and they all work as planned. I'm not sure if you looked at the "Thick" Clip design, but that is the one I am going for and it worked really well. The only thing I am worried about is my PCB. Does that look good to you?

Thanks,
Jason

**Comment by JasonFischell** [comment]
Update: I am going to change all traces to at least 20 mils. Hopefully I'll be able to print Wednesday, if that works for Matt Brown. I'll send him an email confirming tonight or tomorrow.

**Comment by mlp6** [comment]
Yep, you still have some 10 mil traces in there that will be too narrow for the photoetch; otherwise I think it looks good!
